charter.md

```diff
@@ -2,92 +2,82 @@
-
-### A note on terminology (Note: context to be removed from charter upon adoption)
+In some contexts, persons may be willing either to partially disclose some values of their attributes or to demonstrate some properties about their attributes without disclosing their values.
+When received by an entity, a proof of ownership of the digital credential needs to be provided and verified, so that only the legitimate owner of the digital credential can take an advantage of its possession. The use of Trusted Applications (TAs) supported by a Trusted Execution Environment (TEE) is essential in order to provide a proof of ownership of a digital credential.
+Some persons may wish to carry more that one credential. These credentials, together with associated key material, can be stored in an identity digital wallet.
 
-The terms "digital credential", and "digital presentation" are a generic industry terms, which do not imply a specific serialization. 
-
-The terms "verifiable credential", and "verifiable presentation" are terms which can imply, for example, JSON, JSON-LD or mDoc serialization.
+## Background
 
-In the W3C, "verifiable credential", and "verifiable presentation" imply specific JSON-LD based media types.
+W3C has introduced a 'Three Role Model' in the 'Verifiable Credentials Data Model v2.0' specification. VCDM 2.0 (https://w3c.github.io/vc-data-model)
+with the following roles: a holder, an issuer and a verifier.
 
-A "digital credential" is a "W3C Verifiable Credential" when it secures a JSON claimset that conforms to the W3C Technical Recommendation.
-A "digital presentation" is a "W3C Verifiable Presentation" when it secures a JSON claimset that conforms to the W3C Technical Recommendation.
+The wording "Verifiable Credential" (VC) is used in VCDM 2.0. It implies the use of a schema defined using JSON-LD.
+The wording "Verifiable Presentation" (VP) is used in VCDM 2.0. It implies the use of data serialization using JSON-LD.
 
-A "digital credential" is an "OAuth Verifiable Credential" when it secures a JSON claimset that conforms to the requirements of documents developed by the OAuth WG. 
-A "digital presentation" is an "OAuth Verifiable Presentation" when it secures a JSON claimset that conforms to the requirements of documents developed by the OAuth WG. 
+In this charter, the VC concept is captured using the wording "digital credential", while the VP concept is captured using the wording "digital presentation".
 
-A "digital credential" might be a "mDoc verifiable credential", in other specifications.
-A "digital presentation" might be a "mDoc verifiable presentation", in other specifications.
+Some sets of claim names have been defined by the IANA and originate both from the IETF and the OIDF.
+The W3C has taken a different approach allowing to define any kind of claim names by referencing URNs using LD-JSON schemas that include the @context property.
```

Comment:
```suggestion
```

Reply:
This is not an accurate statement, and I am not sure we need to repeat what other working groups outside of IETF are doing, beyond what you have said above.